tests/child_process/host/host.cpp

```diff
@@ -90,7 +92,8 @@ int main(int argc, const char* argv[])
             if (pid == 0) // child process
             {
                 result_child_process = oe_terminate_enclave(enclave);
-                OE_TEST(result_child_process == OE_INVALID_PARAMETER);
+                OE_TEST(result_child_process == OE_OK);
```

Comment:
Given that the enclave is terminated in the parent and the child, I wonder if there is a race condition there. An OE enclave may not be able to be destroyed from both the parent and the child, I'm not sure.

That could be the cause of the build failure.

Reply:
I verified the case on VM or FLC platform and didn't see any error with destroying.
And from the bors log, it shows the error is at Test failed at child process status check: 
> ../tests/child_process/host/host.cpp(113): main child_status == 0

So I guess it needs an "_exit(EXIT_SUCCESS);"  for the child process. I have added it and will start a bors try.

Reply:
oh, ya, that makes sense to me. Thanks!